drivers/battery/sbs_gauge/sbs_gauge.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2021 Leica Geosystems AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_SBS_GAUGE_H_
+#define ZEPHYR_DRIVERS_SENSOR_SBS_GAUGE_H_
+
+#include <stdint.h>
+#include <zephyr/drivers/i2c.h>
+
+/*** Standard Commands ***/
+#define SBS_GAUGE_CMD_MANUFACTURER_ACCESS   0x00 /* ManufacturerAccess */
+#define SBS_GAUGE_CMD_REM_CAPACITY_ALARM    0x01 /* LowCapacityAlarmThreshold */
+#define SBS_GAUGE_CMD_REM_TIME_ALARM        0x02 /* RemainingTimeToEmptyThreshold */
+#define SBS_GAUGE_CMD_BATTERY_MODE          0x03 /* BatteryOperatingMode */
+#define SBS_GAUGE_CMD_AR                    0x04 /* AtRate */
```

Comment:
Standard commands 0x0 to 0x4 are R/W. I don’t see a sbs_cmd_reg_write function in sbs_gauge.c.

Reply:
This PR is focused on adding a fuel-gauge API while maintain feature parity with the sensor version of the sbs_gauge. But you're correct this was a mistake in the original sbs gauge driver. I'll add a todo here to add an sbs_gauge write.

Reply:
This will also be part of an API extension in a future PR to allow some generic writing as well.